android/shared/src/commonMain/kotlin/com/yeobaek/feature/home/HomeViewModel.kt

```
@@ -0,0 +1,88 @@
+                            groupId = it.clubId,
+                            uri = "https://jasdc.or.kr/pub/site/psndc/images/sub/gtop_01.jpg",
+                            title = it.book.title,
+                            groupName = it.name,
+                            groupCount = it.memberCount,
+                        )
+                    },
+                    screenState = ScreenState.Success,
+                )
+            } catch (e: CancellationException) {
+                throw e
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)